src/main/ts/args/Parser.ts

```diff
@@ -24,7 +24,7 @@ const statusDescription = 'Prints out status of current directory.';
 
 const gitUrlOptions: yargs.Options = {
   type: 'string',
-  default: null,
+  'default': null,
```

Comment:
Why is this the opposite to all of the other quote changes? I get that default is a keyword, but also it apparently worked without quotes before so I don't see how the quotes are needed

Reply:
I turned on the setting that requires it for reserved words. It does work, but I've had feedback from engineers that they're uncomfortable with unquoted reserved words as property names.